src/aura/contactList/contactList.css

```diff
@@ -1,7 +1,30 @@
 .THIS .slds-card {
-    margin: 10px;
+    display: inline-block;
+    margin-right:10px;
+    margin-top: .75rem;
 }
-.THIS .custom-card p{
+.THIS .slds-card:before {
+    margin-left:-10px;
```

Comment:
Never, use negative margins, there should be a better solution for that, and btw I don't see this being used